kodilla-spring-basic/build.gradle

```diff
@@ -25,8 +25,10 @@ repositories {
 
 dependencies {
     implementation('org.springframework.boot:spring-boot-starter')
+    implementation 'org.jetbrains:annotations:23.0.0'
```

Comment:
Co to?

Reply:
W moim IntelliJ nie ma możliwości powtarzania wielokrotnego testów. Miałem zapytać na spotkaniu jak to naprawić, ale nie zdążyłem.
Wcześniej próbowałem to naprawić sam i zapomniałem przywrócić ten plik do wersji pierwotnej.